### DISCUSSION TITLE
Is there a way to make CKEditor plugins work?

### DISCUSSION THREAD

**Discussion by liuxiawei** [TriliumNext/Trilium#4469] [Q&A]
How can I enable and use plugins in CKEditor? For example, like [Prism](https://ckeditor.com/cke4/addon/prism), I want to use its code block feature with line numbering. What can I do ?

**Comment by rodrigoalexandrecosta** [comment]
I'm also interested in that, not necessarily Prism but the possibility of using CKEditor plugins.

**Comment by zadam** [comment]
The linked plugin is for CKEditor 4, it won't work in CKEditor 5.

I think recently CKE5 added some support for dynamically loading plugins without rebuilding, but I haven't tried it / explored it: https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/alternative-setups/dll-builds.html
